website/src/components/Footer.js

```diff
@@ -17,22 +56,91 @@ export default function Footer() {
       )}
     >
       <div className="container">
-        <div className={cx('row', styles.nav)}>
+        {!hideNav && footerNav.length > 0 && (
+          <div
+            className={cx([
+              styles.primary,
+              'row row-cols-2 row-cols-lg-auto',
+            ])}
+          >
+            {footerNav.map((item, i) => (
+              <div key={i}>
+                <Link href={item.path}>
+                  <a className={styles.topLevel}>{item.label}</a>
+                </Link>
+                {item.children && item.children.length > 0 && (
+                  <ul className="list-unstyled">
+                    {item.children.map((item, i) => (
+                      <li key={`primary-${i}`}>
```

Comment:
We aren't using named keys for secondary menus - is this intensional? keys are unique under whatever .map they live under and not for the entire doc so it's okay for this to just be `key={i}`